webapp/src/index.tsx

```diff
@@ -327,7 +326,7 @@ export default class Plugin {
 
                 // following the thread only on join. On call start
                 // this is done in the call_start ws event handler.
-                if (profilesInCallInChannel(store.getState(), channelId).length > 0) {
+                if (channelHasCall(store.getState(), channelId)) {
```

Comment:
The idea behind these refactorings are to move logic (like length tests) into the selectors, and make the selector's name declare what it does. This means more selectors, but it's clearer and the logic is isolated to one place. 
Then, if we decide that counting participants is more correct, we can make that change in the selector.